draft-ietf-taps-impl.md

```diff
@@ -135,20 +134,20 @@ Listener objects are created with a Preconnection, at which point their configur
 
 # Implementing Pre-Establishment
 
-The Pre-Establishment phase allows applications to specify properties for the Connections that they are about to make, or to query the API about potential Connections they could make.
+The pre-establishment phase allows applications to specify properties for the Connections that they are about to make, or to query the API about potential Connections they could make.
```

Comment:
"Pre-Establishment" is capitalized in the architecture doc, btw. We may want to change for consistency.